scripts/rmc_flags.py

```diff
@@ -179,6 +179,16 @@ def add_visualizer_flags(make_group, add_flag, config):
                   this is generally the location from which rmc is currently being invoked
                   """)
 
+# Add flags needed for toggling and switching between outputs.
+def add_output_flags(make_group, add_flag, config):
+    group = make_group("Output flags", "Toggle between different styles of output")
```

Comment:
I suggest you to replace these flags by the one command options, such as: `--output-format [old | regular | terse]`. You can use https://docs.python.org/3.6/library/argparse.html#choices to specify the options.

Reply:
Yes, refactoring a lot of code to use Enums and Choices.